File: tfa-test-data-java/AppSecrets.java
package com.tfa.scanner.fixture;

import java.util.Map;

public final class AppSecrets {
    private static final String FALLBACK_OPENAI = "sk-proj-TFA2026JavaFixtureOpenAIA1b2C3d4E5f6G7h8I9j0KLMNOPQRST";
    private static final String JWT_SECRET = "tfa-java-jwt-signing-secret-2026";
    private static final String MYSQL_PASSWORD = "ProdLikeJavaDbPass!2026";

    private AppSecrets() {}

    public static Map<String, String> load() {
        return Map.ofEntries(
            Map.entry("openai.api.key", System.getenv().getOrDefault("OPENAI_API_KEY", FALLBACK_OPENAI)),
            Map.entry("anthropic.api.key", "sk-ant-api03-TFA2026JavaFixtureAnthropicA1b2C3d4E5f6G7h8"),
            Map.entry("google.ai.api.key", "AIzaSyD-TFA2026JavaFixtureGeminiKey8y7x6w5v4u3"),
            Map.entry("azure.openai.key", "1a3c5e7f9b2d4f6a8c0e1b3d5f7a9c2e"),
            Map.entry("azure.openai.endpoint", "https://tfa-java-openai.openai.azure.com/"),
            Map.entry("cohere.api.key", "cohere_TFA2026_java_fixture_key_4917ac8842218c3bd0"),
            Map.entry("mistral.api.key", "mistral_TFA2026_java_fixture_live_23e8d41f5c6a7b2d9f"),
            Map.entry("huggingface.token", "hf_TFA2026JavaFixtureTokenAB12cd34EF56gh78IJ90kl"),
            Map.entry("groq.api.key", "gsk_TFA2026JavaFixtureGroqKeyAB12cd34EF56gh78IJ90"),
            Map.entry("together.api.key", "together_TFA2026_java_fixture_0123456789abcdef"),
            Map.entry("replicate.api.token", "r8_TFA2026JavaFixtureReplicateToken1234567890abcd"),
            Map.entry("github.pat", "github_pat_11JAVAEECCDDEEFF00112233445566778899aabbccddeeff"),
            Map.entry("slack.bot.token", "xoxb-654321987012-TFA2026JAVABOT-abcdefghijklmnopqrstuvwx"),
            Map.entry("aws.access.key.id", "AKIA7TFAJAVA26QWER"),
            Map.entry("aws.secret.access.key", "pQ8wRm3xT7rV2yB9uC1dE6fG0hJ4kL5sA2zX8vB"),
            Map.entry("jwt.secret", JWT_SECRET),
            Map.entry("mysql.password", MYSQL_PASSWORD)
        );
    }
}

File: tfa-test-data-kotlin/AppSecrets.kt
package com.tfa.scanner.fixture

object AppSecrets {
    val values: Map<String, String> = mapOf(
        "openai.api.key" to (System.getenv("OPENAI_API_KEY") ?: "sk-proj-TFA2026KotlinFixtureOpenAIA1b2C3d4E5f6G7h8I9j0KLMNOP"),
        "anthropic.api.key" to "sk-ant-api03-TFA2026KotlinFixtureAnthropicA1b2C3d4E5f6G7h",
        "google.ai.api.key" to "AIzaSyD-TFA2026KotlinFixtureGeminiKey3t2s1r0q9p8",
        "azure.openai.key" to "9c7e5a3d1f8b6d4a2c0e7f5b3d1a9c7e",
        "cohere.api.key" to "cohere_TFA2026_kotlin_fixture_key_0a017ac8842218c3ac6",
        "mistral.api.key" to "mistral_TFA2026_kotlin_fixture_live_c3e8d41f5c6a7b2d9a",
        "huggingface.token" to "hf_TFA2026KotlinFixtureTokenAB12cd34EF56gh78IJ90",
        "groq.api.key" to "gsk_TFA2026KotlinFixtureGroqKeyAB12cd34EF56gh78IJ90",
        "together.api.key" to "together_TFA2026_kotlin_fixture_0123456789abcdef",
        "replicate.api.token" to "r8_TFA2026KotlinFixtureReplicateToken1234567890ab",
        "github.pat" to "github_pat_11KTAABBCCDDEEFF00112233445566778899aabbccddeeff",
        "slack.bot.token" to "xoxb-258147369000-TFA2026KOTLINBOT-abcdefghijklmnopqrstuvwx",
        "aws.access.key.id" to "AKIA7TFAKT26YHNU",
        "aws.secret.access.key" to "vR9wWs4xX8vG3yH1uJ6kL0mN4oP5qR2sA7zX8vB",
        "jwt.signing.secret" to (System.getenv("JWT_SIGNING_SECRET") ?: "tfa-kotlin-jwt-signing-secret-2026"),
        "postgres.password" to "ProdLikeKotlinDbPass!2026"
    )
}
